aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp

```diff
@@ -197,3 +204,131 @@ Tensor ConvertToChannelsLast3dTensor(const Tensor& src) {
 } // namespace at
 
 #endif // USE_FBGEMM
+
+template <int kSpatialDim = 2>
+torch::jit::class_<ConvPackedParamsBase<kSpatialDim>> register_conv_params() {
+  using SerializationType = std::tuple<
+      at::Tensor,
+      c10::optional<at::Tensor>,
+      // these are meant to be torch::List<int64_t> but
+      // it's not supported by onnx, so we'll use Tensor as
+      // a workaround
+      torch::List<at::Tensor>,
+      torch::List<at::Tensor>,
+      torch::List<at::Tensor>,
+      at::Tensor>;
```

Comment:
right, the next bc breaking change would be change from torch::List<at::Tensor> to torch::List<int64_t> when onnx is fixed, do you know how we can make backward compatible changes at that point? I feel it might be easier to separate the arguments instead of putting them together?